src/app/components/PercentageGain/index.tsx

```diff
@@ -13,6 +13,9 @@ interface PercentageGainProps {
  * Negative percentage shows red box with down arrow.
  */
 export const PercentageGain: FC<PercentageGainProps> = ({ earliestValue, latestValue }) => {
+  // If there is no previous data, we can't divide with it, it would just get us a Nan...
+  if (!earliestValue) return
```

Comment:
`earliestValue === 0` would make it easier to see "division by zero" issue

Reply:
OK updated.